tox.ini

```diff
@@ -173,9 +232,9 @@ max-line-length = 88
 
```

Comment:
@tyll, is `[pylint]` section really necessary as we are using `pylintrc`? And what about `[pycodestyle]`?

Reply:
I merge the content of `[pylint]` section with `pylintrc` and remove `[pylint]` and `[pycodestyle]` from `tox.ini` (I suppose `[pycodestyle]` is redundant as `pycodestyle` linter is not used).

Reply:
`[pycodestyle]` and `[pylint]` sections were returned back to `tox.ini` as some IDEs may rely on them.